doc/source/drivers/vector/vrt.rst

```diff
@@ -247,10 +247,10 @@ layer name, and may have the following subelements:
 
 
 - **SrcRegion** (optional) : This element is used to
-  define an initial spatial filter for the source features. This spatial
+  define an initial spatial filter for the source features, such that only features intersecting ``SrcRegion`` will be returned be included in the VRT layer. This spatial
   filter will be combined with any spatial filter explicitly set on the
```

Comment:
I think either "will be returned" or "will be included", not both

Reply:
fixed, thanks